### ISSUE TITLE
Updating a details panel property sometimes doesn't register

### ISSUE THREAD

**Issue by jetXS** [TouchDesigner/TouchEngine-UE#111]
Use Sample 09 to reproduce.
"Unfold" The center property, try to update one of it's members using the mouse / arrow
it will only register and pass the change to TouchEngine when the mouse is released

![image](https://user-images.githubusercontent.com/10253645/142927761-297d3253-ddfb-45b6-a91e-0d32a73ad236.png)

**Comment by ddthejj** [comment]
https://udn.unrealengine.com/s/question/0D54z00007JU38YCAT/difference-between-details-panel-vector-header-vs-children-values

Question asked here

**Comment by ddthejj** [comment]
This issue can be fixed, but only with an engine change - waiting on response from epic with issue ticket

**Comment by jetXS** [comment]
Any update from Epic ?

**Comment by ddthejj** [comment]
> Any update from Epic ?

None - I just bumped the issue, hopefully can get a response soon

**Comment by ddthejj** [comment]
This change will be integrated into Unreal Engine 5.0 - I presume this means it won't be fixed in 4 at all :/

**Comment by jetXS** [comment]
Let's revisit w/ UE5 / put it on hold for now then, thanks

**Comment by jetXS** [comment]
This is fixed in UE5.
